### PR TITLE
Gql01 user create ap itests bird

### PR DESCRIPTION
**Pull request by olgatbird**
userCreate API tests

### CODE REVIEW

File: src/modules/user/_test/userCreateBir.spec.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            const postData = {
+                query: userCreateM,
+                variables: arg
+            }
+            requestGql(postData)
+                .expect(200)
+                .end((err, res) => {
+                    if(err) return done(err);
+                    const respData = res.body.data
+                    console.log("RESP BODY ====", respData)
```
Comment (rustamkutovyi) [inline src/modules/user/_test/userCreateBir.spec.js:19]: Please delete console.log

File: src/modules/user/_test/userCreateBir.spec.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            const postData = {
+                query: ``,
+                variables: arg
+            }
+            requestGql(postData)
+                .expect(400)
+                .end((err, res) => {
+                    if(err) return done(err);
+                    const respData = res.body.errors[0]
+                    console.log("====", respData)
```
Comment (rustamkutovyi) [inline src/modules/user/_test/userCreateBir.spec.js:39]: Please delete console.log

File: src/modules/user/_test/userCreateBir.spec.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            const postData = {
+                query: {userCreateM},
+                variables: arg
+            }
+            requestGql(postData)
+                .expect(400)
+                .end((err, res) => {
+                    if(err) return done(err);
+                    const respData = res.error
+                    console.log("second ====", respData)
```
Comment (rustamkutovyi) [inline src/modules/user/_test/userCreateBir.spec.js:57]: Please delete console.log

**Review by rustamkutovyi — CHANGES_REQUESTED** [review]

File: src/modules/user/_test/userCreateBir.spec.js (hunk 1 above)
Comment (olgatbird) [reply src/modules/user/_test/userCreateBir.spec.js]: Rustam,  thanks, I deleted console.log

File: src/modules/user/_test/userCreateBir.spec.js (hunk 2 above)
Comment (olgatbird) [reply src/modules/user/_test/userCreateBir.spec.js]: Done

File: src/modules/user/_test/userCreateBir.spec.js (hunk 3 above)
Comment (olgatbird) [reply src/modules/user/_test/userCreateBir.spec.js]: Done

Comment (rustamkutovyi) [reply src/modules/user/_test/userCreateBir.spec.js]: Hello Olga! Your code looks good now!

**Review by rustamkutovyi — APPROVED** [review]

File: src/modules/user/_test/userCreateBir.spec.js (hunk 3 above)
Comment (olgatbird) [reply src/modules/user/_test/userCreateBir.spec.js]: Thank you! So, can you merge it?

Comment (olgatbird) [reply src/modules/user/_test/userCreateBir.spec.js]: Okay! I resolved them

**Comment by rustamkutovyi** [comment]
Olga, looks like you merged main into your branch.

**Pull request merged**